presto-main/src/main/java/com/facebook/presto/sql/analyzer/StatementAnalyzer.java

```diff
@@ -917,10 +919,11 @@ protected Scope visitQuerySpecification(QuerySpecification node, Optional<Scope>
             // to pass down to analyzeFrom
 
             Scope sourceScope = analyzeFrom(node, scope);
+            Scope sourceScopeWithWindow = createWindowScope(node, Optional.of(sourceScope));
```

Comment:
As mentioned https://github.com/prestodb/presto/pull/8307#discussion_r124896455 and https://github.com/prestodb/presto/pull/8307#discussion_r134628686, I think we shouldn't be passing the window scope in the subquery. That's why.